tracing/trace.go

```diff
@@ -91,8 +118,8 @@ func (t *trace) toZipkinSpan() ZipkinSpan {
 	zs.Duration = timebp.DurationMicrosecond(end.Sub(t.start))
 
 	var endpoint ZipkinEndpointInfo
-	if t.tracer != nil {
-		endpoint = t.tracer.endpoint
+	if tracer := t.internalTracer(); tracer != nil {
```

Comment:
```suggestion
	if tracer, ok := t.internalTracer(); ok {
```

(see comment above about returning a bool too)